### DISCUSSION TITLE
Proje destek

### DISCUSSION THREAD

**Discussion by Abdullahoda** [IBAX-io/go-ibax#31] [Rewards]
Ibax means mountain goat in Greek. I designed two logos for you, of course, your choice is mine, just a suggestion, your experts will do better
![IMG_34678899](https://user-images.githubusercontent.com/97207588/148708243-dbbab487-f62e-47ba-bbe8-141e9318deed.jpg)
![IMG-35885988](https://user-images.githubusercontent.com/97207588/148708250-2d166fa9-86e5-4a65-80aa-707a1f849710.jpg)

**Comment by HalloKyle** [comment]
WoW, that's a good knowledge to know，Do you mind if we share this one on our Twitter？

**Reply by Abdullahoda** [reply]
Yeah, of course! I'm waiting my gift on Twitter :) id : Piroglu3841

**Reply by emrxsa** [reply]
Hs

**Reply by Metinyucel** [reply]
Muhteşem..

**Reply by Vedat04** [reply]
Sana varya helal olsun..

**Reply by hcaslann** [reply]
Çok iyi

**Reply by Abdullahoda** [reply]
Eyvallah..

**Reply by Selman38** [reply]
Cok guzel bir calisma olmus..

**Reply by Abdullahoda** [reply]
Tskler..

**Reply by Halit38** [reply]
Güzel çalışma tebrikler..

**Reply by Abdullahoda** [reply]
Tskler..

**Reply by frknozdmir** [reply]
Dağ keçisi ha bu harika düşünülmüş😀

**Reply by Abdullahoda** [reply]
Tskler kardesım..

**Comment by Abdullahoda** [comment]
Yeah, of course! I'm waiting my gift on Twitter :) id : Piroglu3841

10 Oca 2022 Pzt 10:37 tarihinde Kyle ***@***.***> şunu yazdı:

**Reply by Abdullahoda** [reply]
?

**Comment by Emirxt** [comment]
Gerçekten güzel bi tasarım olmuş elinize sağlık

**Reply by Abdullahoda** [reply]
Tskler..

**Comment by hemresenol** [comment]
It can be improved a little more. Good luck

**Reply by Abdullahoda** [reply]
Thanks

**Comment by Abdullahoda** [comment]
500 puan için teşekkürler..

**Comment by emrxsa** [comment]
Tsk

**Comment by uludgn71** [comment]
good project

**Reply by Abdullahoda** [reply]
Tskler

**Comment by kantarbozan** [comment]
çok iyi devam böyle güzel olmuş.

**Comment by cagatayakgul** [comment]
Emeklerine sağlik

**Reply by Abdullahoda** [reply]
Tskler.

**Comment by mrtgezgin** [comment]
güzel olmuş. eline sağlık

**Reply by Abdullahoda** [reply]
Tskler

**Comment by Abdullahoda** [comment]
En azından buna ibax ödülü verilebilirdi.:(
